src/views/CreateAccount.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+    <div class="create-account">
+        <div class="wrapper">
+            <b-card no-body img-src="https://picsum.photos/600/300/?image=25" img-alt="Image" img-top tag="article"
+                style="max-width: 30rem;" class="mb-2">
+
+                <b-card-body>
+                    <b-card-title>Profile details: </b-card-title>
+                    <b-card-text>
+                        <table>
+                            <tr>
+                                <td><b>First name: </b></td>
+                                <td>
+                                    <b-form-input id="firstName" v-model="firstName"
+                                        placeholder="Enter your first name"></b-form-input>
+                                </td>
+                            </tr>
```

Comment:
Looks great! No need for table, bootstrap has a <b-container> that is much better. It lets you use grids, style them with bootstrap classes, and easy to make them adjustable. Also, there is a much cleaner way to group fields and labels, check my PR **Profile Details page**